bin/compile

```diff
@@ -36,16 +36,19 @@ log "Generate requirements.txt with Poetry"
 
 log "Install Poetry"
 
-curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py \
+curl -sSL https://install.python-poetry.org \
     | sed -e 's/allowed_executables = \["python", "python3"\]/allowed_executables = ["python3"]/' \
     | python3 \
     | indent
 
-PATH_TO_POETRY_ENV="$HOME/.poetry/env"
+log "Add poetry to the PATH"
+export PATH="/app/.local/bin:$PATH"
```

Comment:
Why this change? Is it no longer working in new Poetry?

Reply:
It was not working with 1.2.0. When I removed it though I wasn’t thinking about past versions… my guess is that the new installer puts them all in the same place now? Sounds Iike people have been testing older versions and not had an issue with it being gone.

Reply:
@felix11h did you have a look how this works in 1.1 / 1.2?